### DISCUSSION TITLE
Add Android versions section

### DISCUSSION THREAD

**Discussion by MuerteRoja** [Droid-ify/client#720] [Ideas]
Hello, good afternoon, a while ago I made a post in the "Issues" section which was deleted (I don't know why, they didn't give me any kind of explanation) so I will proceed to do it in the "Discussion/Ideas" section. I would like to propose that in Droid-Ify, when browsing between applications, in the "Update versions" section under each update it appears which version of Android it supports, as in the F-Droid application, I consider that it would be quite useful for other users

![Imagen de WhatsApp 2024-04-25 a las 13 59 19_01203676](https://github.com/Droid-ify/client/assets/126993006/151e114b-c755-435e-be6d-f3fa050dff07)
![Imagen de WhatsApp 2024-04-25 a las 13 59 19_e8c4e721](https://github.com/Droid-ify/client/assets/126993006/ab908d12-fbc8-4adc-a4a3-e5fb8eda1291)

**Comment by ghost** [comment]
Sounds very useful, indeed! I hadn't even realized this functionality was missing.

**Comment by Iamlooker** [comment]
Will do in next release

**Reply by MuerteRoja** [reply]
> Will do in next release

Well... as you told me last time I took you at your word, we are already on the next update and you still haven't added the feature mentioned above.
![Imagen de WhatsApp 2025-01-19 a las 08 17 20_cff5715c](https://github.com/user-attachments/assets/506bc283-6e24-4be8-9641-901a817fcd8e)
![Imagen de WhatsApp 2025-01-19 a las 08 17 20_e42c226d](https://github.com/user-attachments/assets/8e93eafc-5599-431b-9066-304cead1c836)
![Imagen de WhatsApp 2025-01-19 a las 08 17 20_4b1fc9c6](https://github.com/user-attachments/assets/9b3d6d13-c9f2-4806-a113-857178355313)
![Imagen de WhatsApp 2025-01-19 a las 08 17 20_f3545af3](https://github.com/user-attachments/assets/e90c4ee3-011d-48b4-8ccb-d18ff8f23723)

**Reply by Iamlooker** [reply]
I cannot remember everything 😅. I am a solo dev after all, I will try and include it in the next update(If I am able to remember, sir)

**Reply by MuerteRoja** [reply]
> I cannot remember everything 😅. I am a solo dev after all, I will try and include it in the next update(If I am able to remember, sir)

And the next update might come out in 6 or 12 months and again you might forget about it, due to the latest events I prefer to switch to using the F-Droid application, I consider it a shame, I mean, I really liked the comfort of the Droid-Ify interface, and, from my point of view (and from the point of view of the rest of the people who have commented on this forum) it would have been great to add this feature, but well, I understand the circumstances, judging by how you tell me I would say that Droid-Ify is only being maintained by one person, in which case I can understand it, and, as I am not in a position to ask for pears from an elm tree (it is a saying that we have in Spain) I will simply change the application, greetings, have a nice day Mr. developer

**Comment by thepragmaticmero** [comment]
feet

**Comment by Iamlooker** [comment]
885c09a2bb86f0bc47e3566f53c8dc35384393a5
